patches/54540830 - The Bigs 2.patch.toml

```
@@ -5,7 +5,7 @@ hash = "5187638D00D4A75A" # default.xex
 
 [[patch]]
     name = "Enable Debug Menu"
-    desc = "Required for Debug Settings and Cheats."
+    desc = "Required for Debug Settings and Cheats. Hold start/select in the main menu."
```

It should be `Start+Select` if both need to be held at the same time.
```suggestion
    desc = "Required for Debug Settings and Cheats. Hold Start+Select in the main menu."
```